avocado/plugins/runner_nrunner.py

```diff
@@ -258,6 +258,8 @@ def run_suite(self, job, test_suite):
         self.runtime_tasks = self._get_all_runtime_tasks(test_suite)
         if test_suite.config.get('nrunner.shuffle'):
             random.shuffle(self.runtime_tasks)
+        test_ids = [rt.task.identifier for rt in self.runtime_tasks
+                    if rt.task.category == 'test']
```

Comment:
Ohhh, I was also concerned about `rt` initially. But since that now we have `self.runtime_tasks` (what is good), rt here makes much more sense.